test/integration/scenarios/10.namespace/verify/manifest_test.go

```diff
@@ -58,17 +58,19 @@ func TestBasic_ReadNondefaultNamespaceActivation(t *testing.T) {
 		err = json.Unmarshal(bytes, &state)
 		require.NoError(t, err)
 
-		status := state.Status.Status
-		fmt.Printf("Current activation status: %s\n", status)
-		if status == v1alpha2.Done {
-			require.Equal(t, 2, len(state.Status.StageHistory))
-			require.Equal(t, v1alpha2.Done, state.Status.StageHistory[0].Status)
-			require.Equal(t, v1alpha2.Done, state.Status.StageHistory[1].Status)
-			require.Equal(t, 3, len(state.Status.StageHistory[1].Inputs))
-			require.Equal(t, 10., state.Status.StageHistory[1].Inputs["age"])
-			require.Equal(t, "worker", state.Status.StageHistory[1].Inputs["job"])
-			require.Equal(t, "sample", state.Status.StageHistory[1].Inputs["name"])
-			break
+		if state.Status != nil {
+			status := state.Status.Status
```

Comment:
Can we merge the test to 05?

Reply:
I tested in 05 before, but it interfered with the original test. So I moved it out to keep the test env clean